charts/kubeit-app-of-apps/templates/applications.yaml

```diff
@@ -59,6 +59,15 @@
 {{- end }}
 {{- end }}
 
+# Create a map with Helm Values inline
+{{ $allValues := dict }}
+{{- range $filename := $valuesList -}}
+  {{ $allValues = mergeOverwrite $allValues ($.Files.Get . | fromYaml) }}
```

Comment:
Check if valuesFiles are accessiable when app of apps chart is referenced as subchart